sass/style.scss

```diff
@@ -55,6 +55,18 @@ a {
   color: white;
 }
 
+.ie-fix-vertical, div.col-8, div.col-10, div.col-12, div.maketitle {
```

Comment:
I don't have IE but looks good to me in Brave. My one question is whether this col modification does anything for other places that use these classes?

Reply:
@nburt I believe it was okay, but it's now moot-- that css rule is gone.